### PR TITLE
(feat): Added pod-network-corruption chart

### LINKED ISSUE #1060: (feat): Add chart for pod network packet corruption experiment
**What needs to done**

- Add chaos chart for pod network packet corruption experiment in litmuschaos/chaos-chart repo.

- Add chartServiceVersion for network packet corruption experiment in litmuschaos/chaos-chart repo.

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

- Added pod-network-corruption chart
- fixes: litmuschaos/litmus#1060

**Logs with minimal permission**

[pod-network-corruption.txt](https://github.com/litmuschaos/chaos-charts/files/4030245/pod-network-corruption.txt)

### CODE REVIEW

File: charts/generic/pod-network-corruption/experiment.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+kind: ChaosExperiment
+metadata:
+  name: pod-network-corruption
+  version: 0.1.0
+spec:
+  definition:
+    permissions:
+      apiGroups:
+        - ""
+        - "extensions"
```
Comment (ksatchit) [inline charts/generic/pod-network-corruption/experiment.yaml]: minimal required, scope

File: charts/generic/pod-network-corruption/pod-network-corruption.chartserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    Pod-network-corruption contains chaos to disrupt network connectivity to kubernetes pods. Experiments can inject percentage packet corruption on the app replica pods. 
+    - Causes packet corruption of application replica by injecting packet corruption using pumba.
+    - The application pod should be healthy once chaos is stopped. Service-requests should be served (say, via alternate replicas) despite chaos.
+  keywords:
+    - Kubernetes
+    - Network
+    - Corruption
+  platforms:
+    - GKE
+    - OpenShift
```
Comment (ksatchit) [inline charts/generic/pod-network-corruption/pod-network-corruption.chartserviceversion.yaml]: might want to confirm

File: charts/generic/pod-network-corruption/pod-network-corruption.chartserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  name: pod-network-corruption
+  version: 0.1.0
+  annotations:
+    categories: Kubernetes
+    vendor: CNCF
+    support: https://slack.kubernetes.io/
+spec:
+  displayName: pod-network-corruption
+  categoryDescription: |
+    Pod-network-corruption contains chaos to disrupt network connectivity to kubernetes pods. Experiments can inject percentage packet corruption on the app replica pods.
```
Comment (ksatchit) [inline charts/generic/pod-network-corruption/pod-network-corruption.chartserviceversion.yaml:13]:
```suggestion
    Pod-network-corruption contains chaos to disrupt network connectivity to kubernetes pods. This Experiment will inject packet corruption of desired percentage on the app replica pods. 
```

File: charts/generic/pod-network-corruption/pod-network-corruption.chartserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  version: 0.1.0
+  annotations:
+    categories: Kubernetes
+    vendor: CNCF
+    support: https://slack.kubernetes.io/
+spec:
+  displayName: pod-network-corruption
+  categoryDescription: |
+    Pod-network-corruption contains chaos to disrupt network connectivity to kubernetes pods. Experiments can inject percentage packet corruption on the app replica pods. 
+    - Causes packet corruption of application replica by injecting packet corruption using pumba.
```
Comment (ksatchit) [inline charts/generic/pod-network-corruption/pod-network-corruption.chartserviceversion.yaml:14]:
```suggestion
    - Causes packet corruption on application replica by injecting packet corruption using pumba.
    - Injects network packet corruption on application replicas using pumba
```

File: charts/generic/pod-network-corruption/pod-network-corruption.chartserviceversion.yaml (hunk 2 above)
Comment (shubhamchaudhary) [reply charts/generic/pod-network-corruption/pod-network-corruption.chartserviceversion.yaml]: Yes it is working

Comment (ispeakc0de) [reply charts/generic/pod-network-corruption/pod-network-corruption.chartserviceversion.yaml]: Yes @ksatchit , It is working

File: charts/generic/pod-network-corruption/experiment.yaml (hunk 1 above)
Comment (ispeakc0de) [reply charts/generic/pod-network-corruption/experiment.yaml]: done

**Review by ksatchit — APPROVED** [review]

**Review by chandankumar4 — APPROVED** [review]

**Pull request merged**